tests/DotNet.Testcontainers.Tests/Unit/Containers/Unix/Modules/Databases/MsSqlTestcontainerTest.cs

```diff
@@ -67,7 +67,7 @@ public async Task ExecScriptInRunningContainer()
       Assert.Contains("MyName", result.Stdout);
     }
 
-    [IgnoreOnLinuxEngine]
+    [Fact]
```

Comment:
I changed these attributes as I could not execute these tests under Linux. The `SqlClient` seems to be only supported under windows:
```
System.PlatformNotSupportedException
System.Data.SqlClient is not supported on this platform.
```

Reply:
No, those tests run on Linux too.